libmc/_client.pyx

```diff
@@ -472,7 +494,7 @@ cdef class PyClient:
         cdef bytes py_value = self._get_raw(GET_OP, key2, &flags, &cas_unique)
 
         if py_value is not None and self.do_split and (flags & _FLAG_DOUBAN_CHUNKED):
-            n_splits = int(py_value.strip('\0'))
```

Comment:
`.strip('\0')` here is essential for compatibility with python-libmemcached. ( `\0` is added in `strlen(chunk_key)+1` https://github.com/douban/python-libmemcached/blob/2d2cae5/split_mc.c#L37 )